### DISCUSSION TITLE
static下的静态资源访问  如何隐藏 X-Powered-By 这个请求头呀

### DISCUSSION THREAD

**Discussion by biggod-ck** [thinkjs/thinkjs#1714] [Q&A] [answered]
static下的静态资源访问  如何隐藏 X-Powered-By 这个请求头呀

**Comment by lizheming** [comment, marked as answer]
static 直接在 nginx 那边拦截请求不走 ThinkJS 就好了

**Comment by zhangbowy** [comment]
> X-Powered-By

![image](https://user-images.githubusercontent.com/33514252/220574263-2931c415-9b53-4b7f-b3e8-7dc89dd0fb06.png)
隐藏了干哈，又不影响啥

**Comment by zhangbowy** [comment]
> 如何隐藏 X-Powered-By 这个请求头呀
![image](https://user-images.githubusercontent.com/33514252/220575423-8ec94551-2ffa-4f3b-842b-b17180f54a18.png)
https://github.com/thinkjs/think-meta
